R/helpers.R

```diff
@@ -201,30 +201,26 @@ create_directory <- function(base_path, path) {
   target_path
 }
 
-edit_file <- function(base_path, path) {
-  full_path <- path.expand(file.path(base_path, path))
-
-  ## example: path = ".R/snippets/r.snippets" but .R doesn't exist yet
-  if (dirname(path) != ".") {
-    create_directory(base_path, dirname(path))
-  }
+edit_file <- function(path) {
+  full_path <- path.expand(path)
+  create_directory(dirname(dirname(full_path)), basename(dirname(full_path)))
 
   if (!file.exists(full_path)) {
     file.create(full_path)
   }
 
   if (!interactive() || is_testing()) {
-    todo("Edit ", value(path))
+    todo("Edit ", value(basename(path)))
```

Comment:
I will make this & the one below better when working on #177 (Use fs when released) and #198 (Path reporting).